### PR TITLE
Notifications page

### PR DESCRIPTION
**Pull request by itexpert120**
Resolves https://github.com/NEARBuilders/gateway/issues/282

https://github.com/NEARBuilders/gateway/assets/46527178/bc38ab18-4b05-480b-ae11-136fe5281287

### CODE REVIEW

File: apps/builddao/widget/TrialAccountBanner.jsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const Container = styled.div`
 
   img {
     width: 100%;
-    max-height: 100vh;
```
Comment (elliotBraem) [inline apps/builddao/widget/TrialAccountBanner.jsx:23]: Why this change?

File: apps/builddao/widget/app.jsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ const config = {
     "--secondary-color": "white",
     background: "var(--main-color)",
     color: "var(--secondary-color)",
-    height: "100vh",
+    height: "100%",
```
Comment (elliotBraem) [inline apps/builddao/widget/app.jsx:12]: And this?

File: apps/builddao/widget/app/view.jsx [hunk 3]
```diff
@@ -82,6 +82,7 @@ const Container = styled.div`
 const Content = styled.div`
   width: 100%;
   height: 100%;
+  background: black;
```
Comment (elliotBraem) [inline apps/builddao/widget/app/view.jsx:85]: Can we move this to the root?

File: apps/builddao/widget/logout.jsx [hunk 4]
```diff
@@ -11,7 +11,7 @@ const LogoutContainer = styled.div`
 
   img {
     width: 100%;
-    max-height: 100vh;
+    min-height: 100vh;
```
Comment (elliotBraem) [inline apps/builddao/widget/logout.jsx:14]: multiple times... I'm imagining all these images that take up full view height; is this right?

File: apps/builddao/widget/page/notifications.jsx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+        props={item}
+      />
+    </StyledNotification>
+  );
+};
+
+return (
+  <div className="placeholder-glow container-xl mt-3">
+    <Widget
+      src="mob.near/widget/FilteredIndexFeed"
```
Comment (elliotBraem) [inline apps/builddao/widget/page/notifications.jsx:59]: ${alias_mob}

**Comment by Megha-Dev-19** [comment]
@itexpert120 quick question, do we show custom notifications? specially the builddao custom notification?

File: apps/builddao/widget/app.jsx (hunk 2 above)
Comment (elliotBraem) [reply apps/builddao/widget/app.jsx:12]: The concern with this one is that now, in the preview you provided, the height of the page "grows" down as the page loads. With 100vh, the whole screen starts with full height instead

File: apps/builddao/widget/TrialAccountBanner.jsx (hunk 1 above)
Comment (itexpert120) [reply apps/builddao/widget/TrialAccountBanner.jsx:23]: I'll revert this. It was unintentional.

File: apps/builddao/widget/app/view.jsx (hunk 3 above)
Comment (itexpert120) [reply apps/builddao/widget/app/view.jsx:85]: Yes, sure

**Comment by itexpert120** [comment]
> @itexpert120 quick question, do we show custom notifications? specially the builddao custom notification?

We can create custom notification and add support for it in our gateway

File: apps/builddao/widget/page/notifications.jsx [hunk 6]
```diff
@@ -0,0 +1,63 @@
+const accountId = context.accountId;
+
+if (!accountId) {
+  return "Sign in with NEAR Wallet";
+}
```
Comment (Megha-Dev-19) [inline apps/builddao/widget/page/notifications.jsx:5]: should we show a login button to navigate them to `join` page?
Like we do for feed I guess

**Review by Megha-Dev-19 — APPROVED** [review]
Can you please create a ticket to support custom notifications in the feed.
Left a comment
LGTM

File: apps/builddao/widget/page/notifications.jsx (hunk 6 above)
Comment (elliotBraem) [reply apps/builddao/widget/page/notifications.jsx:5]: I think this sounds like a nice addition

**Comment by elliotBraem** [comment]
@itexpert120 If we can resolve merge conflict, add the login button for join page when not logged in, and create the ticket for supporting custom notification for buildhub.near, this looks good to me!

**Comment by itexpert120** [comment]
Resolved
